src/components/modebar/modebar.js

```diff
@@ -283,6 +285,10 @@ proto.hasButtons = function(buttons) {
     return true;
 };
 
+function jsVersion(str) {
+    return str.replace('Plotly', 'Plotly.js') + ' (v' + version + ')';
```

Comment:
That'll work, as long as translators always include exactly `Plotly` in their output. I still think it would be better to change to `Plotly.js` in the translation files, partly because translators may look at the output while making these translation, so might be puzzled that they write `由Plotly生成` but get `由Plotly.js生成` - or they might write `由Plotly.js生成` and get `由Plotly.js.js生成` 🙀 . Also just in case for whatever reason the `.js` affects how the translation should be written.

Reply:
OK. Addressed in https://github.com/plotly/plotly.js/pull/6077/commits/70e25d9312c60846f2736564b7206db81e0aa164.

Reply:
Thx - sorry for being picky 😅